person.rb

```diff
@@ -22,6 +23,11 @@ def can_use_services?
     @age || @parent_permission
   end
 
+  def add_rentals(rental)
+    @rentals.push(rental)
+    rental.book = self
+  end
```

Comment:
- [ ] [OPTIONAL]
It will be a good idea to optimize our `add_rental` method so that it takes _book_ and _date_ parameters, like the following code example :

```ruby
class Person
    attr_accessor :name, :age, :rentals
  .
  .

  def initialize(age, name = "Unknown", parent_permission = true)
    .
    .
    # *@rentals should be initialized. Link to the example about 
    @rentals = []
  end

  def add_rental(book, date)
        Rental.new(date, book, self)
  end

  .
  .
end
```